src/lightning/pytorch/utilities/combined_loader.py

```diff
@@ -329,6 +343,7 @@ def __next__(self) -> _ITERATOR_RETURN:
         out, batch_idx, dataloader_idx = out
         return tree_unflatten(out, self._spec), batch_idx, dataloader_idx
```

Comment:
Does `__next__` not need it?

Reply:
This class inherits from `collections.abc.Iterable`, which only implements the abstract method `__iter__` (https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes).